core-rules/src/main/java/org/hisp/dhis/rules/functions/RuleFunctionCheckControlDigits.java

```diff
@@ -0,0 +1,37 @@
+package org.hisp.dhis.rules.functions;
+
+import org.hisp.dhis.rules.RuleVariableValue;
+
+import java.util.List;
+import java.util.Map;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import javax.crypto.IllegalBlockSizeException;
+
+final class RuleFunctionCheckControlDigits extends RuleFunction {
```

Comment:
This function do nothing. Does it exist documentation about this function in dhis 2 docs? I have not found it [here](https://docs.dhis2.org/2.25/en/user/html/manage_program_rule.html) and this function is not enable via tracker capture web, only exists not implemented in dhis2.angular.services.js.  Has sense that we adding this empty function to SDK? I think that we should not add it because generate only noise.